src/Open3D/Visualization/Visualizer/GuiVisualizer.cpp

```diff
@@ -1636,6 +1674,20 @@ void GuiVisualizer::SetGeometry(
         impl_->settings_.wgt_point_size->SetEnabled(num_point_clouds > 0);
     }
 
+    if (!impl_->settings_.loaded_materials_.empty()) {
+        int resetIdx = impl_->settings_.wgt_prefab_material->AddItem(
+                MATERIAL_FROM_FILE_NAME.c_str());
+        impl_->settings_.wgt_prefab_material->SetSelectedIndex(resetIdx);
+        ;
```

Comment:
remove empty ;?

Reply:
This crappy butterfly keyboard is driving me nuts. I think my enter key repeated itself. Removed.